### PR TITLE
Club Page no data behavior

### PR DESCRIPTION
**Pull request by steven-solar**
This PR fixes the club rendering in the case that there are no members, no book or no assignments. Currently, as short circuiting uses clauses like `this.state.members.length && <____>`, in the case that there were no members/assignments/book it would render a "0". This PR changes this behavior so it outputs useful text instead. 

Screenshot 2020-08-03 at 1 33 10 PM

### CODE REVIEW

**Review by MauroGuerrero20 — APPROVED** [review]
LGTM

**Review by antoniolinhart — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/ClubPage.jsx [hunk 1]
```diff
@@ -117,10 +119,17 @@ class ClubPage extends Component {
 
   render() {
     const isOwner = this.state.owner && this.state.club.ownerID === window.localStorage.getItem('userID');
-    const bookTile = this.state.book.authors && <BookSearchTile book={this.state.book} bookLists={this.props.bookLists} updateBookLists={this.props.updateBookLists} />;
+    const bookTile = this.state.book.authors 
+        ? <BookSearchTile book={this.state.book} bookLists={this.props.bookLists} updateBookLists={this.props.updateBookLists} />
```
Comment (antoniolinhart) [inline capstone-frontend/src/components/ClubPage.jsx]: Could you break up the props into multiple lines to make it more clear?

File: capstone-frontend/src/components/ClubPage.jsx [hunk 2]
```diff
@@ -117,10 +119,17 @@ class ClubPage extends Component {
...
+        : <div> No book yet! </div>
     const owner = this.state.owner && <UserCard removeMember={this.removeMember} club={this.state.club} user={this.state.owner} />;
-    const members = this.state.members.length && this.state.members.map(m => <UserCard key={m.id} user={m} club={this.state.club} removeMember={this.removeMember} />);
-    const assignments = this.state.assignments.length && <div> {this.state.assignments.map(a => <AssignmentCard key={a.id} assignment={a} />)} </div>;
+    const members = this.state.members.length > 0 
+        ? this.state.members.map(m => <UserCard key={m.id} user={m} club={this.state.club} removeMember={this.removeMember} />)
```
Comment (antoniolinhart) [inline capstone-frontend/src/components/ClubPage.jsx]: Same here

**Review by VivekShankar — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/ClubPage.jsx (hunk 1 above)
Comment (steven-solar) [reply capstone-frontend/src/components/ClubPage.jsx]: Sure!

**Pull request merged**